_src/static/sass/utilities/_variables.scss

```diff
@@ -193,34 +193,31 @@ $font-family-monospace: 'Bitstream Vera Sans Mono', Consolas, Courier, monospace
 // typography: config - scale
 $modular-scale-ratio: $perfect-fourth;
 
-$font-scale-small: 16;
-$font-scale-medium: 18;
+$font-scale-small: 14;
+$font-scale-medium: 16;
 $font-scale-large: 18;
 $font-scale-base: $font-scale-medium;
 
-// typography: sizes
+// typography: sizes (pixels)
 $font-sizes: (
-    base:           $font-scale-base,
-    mid-small:      modular-scale(-1, $font-scale-base),
-    small:          modular-scale(-2, $font-scale-base),
-    x-small:        modular-scale(-3, $font-scale-base),
-    xx-small:       modular-scale(-4, $font-scale-base),
-    mid-large:      modular-scale(1, $font-scale-base),
-    large:          modular-scale(2, $font-scale-base),
-    x-large:        modular-scale(3, $font-scale-base),
-    xx-large:       modular-scale(4, $font-scale-base)
+    xx-large:       43,
+    x-large:        32,
+    large:          24,
+    mid-large:      21,
+    mid-small:      18,
+    base:           $font-scale-base
```

Comment:
@clrux, it looks like the font scale here is off from what was designed in https://openedx.atlassian.net/browse/UX-1524. You may want to check the Sketch file (https://drive.google.com/a/edx.org/folderview?id=0B7jcFrBngjmkTGdpTnNJb29faUE&usp=sharing) and inspect each heading's text properties. 

I'll try to add notes inline where things are off in this map.